### ISSUE TITLE
Undocumented dependencies

### ISSUE THREAD

**Issue by pwarelis** [andyperlitch/angularjs-table#30]
The documentation doesn't mention that this component depends on **angular-ui/ui-sortable** and that depends on **jQuery**. These two significantly bloat the resulting bundles.

Maybe make the sorting optional?

**Comment by andyperlitch** [comment]
Yes, making the sorting functionality sortable sounds fine. I should have some time today or tomorrow to make that change.

Thanks for the suggestion!

**Comment by andyperlitch** [comment]
Yikes, 22 days still... Apologies for the delay, I've been busier than expected. Will attempt to address this soon

**Comment by andyperlitch** [comment]
@pwarelis please have a look at #36 and tell me if that works for you.

**Comment by pwarelis** [comment]
Hey Andy,

What about the module name 'ui.sortable' on angular module boot?

**Comment by andyperlitch** [comment]
Oh good call, will update the PR
